pkgs/development/interpreters/octave/default.nix

```diff
@@ -57,14 +54,14 @@
 , qtbase ? null
 , qtsvg ? null
 , qtscript ? null
+, wrapQtAppsHook ? null
```

Comment:
BTW There's some cleanup to do here regarding the nulls as they're not doing anything because they're filled in automatically with callPackage.

The `optionals` in buildInputs checking for null are unnecessary because if a user overrides something to null then it does not matter if it's in buildInputs.

`null` as default values for attrs in nixpkgs is legacy cruft.

Reply:
> `null` as default values for attrs in nixpkgs is legacy cruft.

Right, thanks!